node/forest_libp2p/src/rpc/behaviour.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use super::handler::RPCHandler;
+use super::RPCEvent;
+use futures::prelude::*;
+use futures::task::Context;
+use futures_util::task::Poll;
+use libp2p::core::ConnectedPoint;
+use libp2p::swarm::{
+    protocols_handler::ProtocolsHandler, NetworkBehaviour, NetworkBehaviourAction, PollParameters,
+};
+use libp2p::{Multiaddr, PeerId};
+use std::marker::PhantomData;
+
+pub struct RPC<TSubstream> {
+    /// Queue of events to processed.
+    events: Vec<NetworkBehaviourAction<RPCEvent, RPCMessage>>,
+    /// Pins the generic substream.
```

Comment:
I don't believe these comments will be visible in our docs as the `RPC` struct is not commented on. If the intent was to have them visible to the docs I think you will need to also document the struct.